### PR TITLE
docs: add content docs

### PR DESCRIPTION
**Pull request by debs-obrien**
docs: add content docs
Todo: awaiting images from Vincent but should have them soon
@ricardogobbosouza please check they are ok. I split options into 2, options and downloading. Let me know what you think.

### CODE REVIEW

File: docs/content/settings.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "title": "Google Fonts",
+  "url": "https://google-fonts@nuxtjs.org",
```
Comment (ricardogobbosouza) [inline docs/content/settings.json:3]: Isn't it https://google-fonts.nuxtjs.org?

**Review by ricardogobbosouza — CHANGES_REQUESTED** [review]

**Comment by ricardogobbosouza** [comment]
@debs-obrien 
I think that separated into two sections is better 😄

File: docs/content/settings.json (hunk 1 above)
Comment (debs-obrien) [reply docs/content/settings.json:3]: Oops good catch will fix

**Comment by debs-obrien** [comment]
@ricardogobbosouza I think it's ready. Might need @pi0s help in setting up the docs domain

**Review by ricardogobbosouza — CHANGES_REQUESTED** [review]
What do you think about simplifying `README.md`
like https://github.com/nuxt-community/pwa-module?

**Comment by debs-obrien** [comment]
Yes. Easier to maintain that way just wasn't sure if it was necessary to have something on the readme or not. But lets keep it simple and we can always change it back if needed. Will amend

**Comment by pi0** [comment]
https://google-fonts.nuxtjs.org/

**Comment by debs-obrien** [comment]
Yeah. Thanks @pi0. @ricardogobbosouza I didn't reduce the readme yet will do that during week

**Pull request merged**